php/class-plugin.php

```diff
@@ -36,7 +36,6 @@ public function __construct() {
 	 */
 	public function init() {
 		$this->config = apply_filters( 'customize_pane_resizer_plugin_config', $this->config, $this );
-		$this->min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
```

Comment:
Take a look at your branch: https://github.com/jtsternberg/wp-customize-pane-resizer/tree/feature/store-in-sessionstorage

Notice:

<img width="277" alt="jtsternberg_wp-customize-pane-resizer_at_feature_store-in-sessionstorage" src="https://cloud.githubusercontent.com/assets/134745/11675373/2347e72c-9dde-11e5-9e26-f5c36e9f10c9.png">

And if you view one of them:

<img width="520" alt="wp-customize-pane-resizer__jscsrc_at_feature_store-in-sessionstorage_ _jtsternberg_wp-customize-pane-resizer" src="https://cloud.githubusercontent.com/assets/134745/11675380/35afa9c2-9dde-11e5-8d40-35798613a6be.png">

So somehow the symlink file is getting corrupted so that JSON string is getting written to the file as if it is the file path the symlink is pointing to. Since this path obviously doesn't exist, the system treats the files as non-existent.

I'll fix after merging.